features-json/web-app-manifest.json

```diff
@@ -296,39 +296,39 @@
       "11.5":"n",
       "12":"n",
       "12.1":"n",
-      "46":"y"
+      "46":"a #1"
     },
     "and_chr":{
       "69":"y"
     },
     "and_ff":{
-      "62":"y #2"
+      "62":"a #1"
     },
     "ie_mob":{
       "10":"n",
       "11":"n"
     },
     "and_uc":{
-      "11.8":"y"
+      "11.8":"a #1"
     },
     "samsung":{
-      "4":"y",
-      "5":"y",
-      "6.2":"y",
-      "7.2":"y"
+      "4":"a #1",
+      "5":"a #1",
+      "6.2":"a #1",
+      "7.2":"a #1"
     },
     "and_qq":{
-      "1.2":"y"
+      "1.2":"a #1"
     },
     "baidu":{
-      "7.12":"y"
+      "7.12":"a #1"
     }
   },
   "notes":"",
   "notes_by_num":{
-    "1":"On Windows, Chrome will read the manifest to generate a desktop icon when selecting the \"Add to desktop...\" option.",
-    "2":"The installation events, `appinstalled` and `beforeinstallprompt`, are not supported.",
-    "3":"The installation events, `appinstalled` and `beforeinstallprompt`, are not supported on all platforms"
+    "1":"The installation events, `appinstalled` and `beforeinstallprompt`, are not supported.",
+    "2":"The installation events, `appinstalled` and `beforeinstallprompt`, are not supported on all platforms."
+    "3":"The manifest may be used to list apps in the Microsoft Store, which will use Edge as a back-end. However, Edge does not directly implement the Manifest spec."
```

Comment:
@patrickkettner, as this is Microsoft specific, could you kindly check this wording to make sure it's accurate?